myuw/templates/handlebars/card/instructor_schedule/course_content.html

```diff
@@ -3,7 +3,7 @@
 
     <div id="instructor_sche_on_course_card{{index}}">
     </div>
-    <div id="instructor_course_resource{{index}}">
+    <div id="instructor_course_resource{{index}}" class="{{#if mini_card}}{{#unless allows_secondary_grading}}{{#if evaluation.eval_not_exist}}last{{/unless}}{{/unless}}{{/if}}">
```

Comment:
Could you sort out the open and close handlebar conditions?
They need to be in matching order.